database/models/users.db.php

```diff
@@ -81,11 +81,16 @@ function set_email($real_email, $nose_email)
 
     function set_perso($sportident, $address, $postal_code, $city, $phone)
     {
-        $this->sportident = $sportident;
+        $this->sportident = $sportident ?: 0;
         $this->address = $address;
         $this->postal_code = intval($postal_code);
         $this->city = $city;
-        $this->$phone = intval($phone);
+        $normalized_phone = preg_replace('/[^0-9]/', '', $phone);
+        $normalized_phone = substr($normalized_phone, -9);
+        $final_phone = '+33' . $normalized_phone;
+        $this->phone = $final_phone;
```

Comment:
If I use my swedish phone here, it's not ideal. Maybe make this a bit more generic, no need for the extra complexity imo.
About the user entity, I think we should simplify a lot:
- remove address, postal code and city
- remove SI (licence number too?)
- add `removed` boolean column for soft delete